rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/OracleTableMetaCache.java

```diff
@@ -30,6 +25,14 @@
 import io.seata.rm.datasource.sql.struct.TableMeta;
 import io.seata.sqlparser.util.JdbcConstants;
 
+import java.sql.Connection;
```

Comment:
don't move imports

Reply:
@a364176773 @slievrly @l81893521
  1, I moved back some imports that i moved down before.
   2,I have added some comments.